### DISCUSSION TITLE
is it possible to get a stored state without a hook on react-native?

### DISCUSSION THREAD

**Discussion by benohiit** [pmndrs/jotai#2538] [Q&A]
Hi,
I am using react-native `atomWithStorage`, and I want to acces the stored data outside of a component. Any idea / example of code ?
Its mainly to get the langage state in the files that init `i18next`

**Comment by dai-shi** [comment]
You can use the Store API.
Either use `<Provider store={store}>` and passing your store, or if it's Provider-less, `getDefaultStore()` will give you the store.

**Reply by benohiit** [reply]
So if I understand well, it will simply look like something like that ?

```
import { getDefaultStore } from "jotai";
import AppAtom from '../../utils/atomState';     //my storage
const store = getDefaultStore();
const atom = store.get(AppAtom);
console.log("stored langage: ", atom.lang)
```
But I recieve .getDefaultStore is undefined
I guess its because of the version (I have the 1.7.4), do you know from which version the defaultStore got aligned ?

**Reply by dai-shi** [reply]
I think it's from v2, or maybe after that. I recommend to update to the latest version anyway.
